modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/extensions/Toolbar.kt

```diff
@@ -67,8 +67,13 @@ fun Toolbar.setup(
             true
         }
     }
+    addStatusBar()
 }
 
 fun Toolbar.setupChromeCastButton(context: Context?, onClick: () -> Unit) {
     menu.setupChromeCastButton(context, onClick)
 }
+
+fun Toolbar.addStatusBar() {
```

Comment:
Can be `private`. And perhaps rename it to something like `includeStatusBarPadding()`?

Reply:
That's a much better name, thanks for the suggestion. I might leave it public as I need for a future PR.